platform/ext/target/nordic_nrf/common/core/attest_hal.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2018-2021, Arm Limited. All rights reserved.
+ * Copyright (c) 2022 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+#include "tfm_attest_hal.h"
+#include "tfm_plat_boot_seed.h"
+#include "tfm_plat_device_id.h"
+#include "tfm_plat_otp.h"
+#include <nrf_cc3xx_platform.h>
```

Comment:
nrf_cc3xx_platform is not available upstream, so we can't actually send this upstream.
That is no problem though. We just need to integrate this in a different way.